frontend/src/scenes/billing/BillingProductAddon.tsx

```diff
@@ -275,14 +275,18 @@ export const BillingProductAddon = ({ addon }: { addon: BillingProductV2AddonTyp
                             </>
                         )}
                     </div>
-                    {!addon.inclusion_only && !addon.trial && isProrated && !addon.contact_support && (
-                        <p className="mt-2 text-xs text-muted text-right">
-                            Pay ~${prorationAmount} today (prorated) and
-                            <br />
-                            {formatFlatRate(Number(upgradePlan?.unit_amount_usd), upgradePlan?.unit)} every month
-                            thereafter.
-                        </p>
-                    )}
+                    {!addon.inclusion_only &&
+                        !addon.trial &&
+                        isProrated &&
+                        !addon.contact_support &&
+                        !billing?.trial && (
```

Comment:
quite the conditional 😅

Reply:
Yeah ... I will go back and clean some of these up.